addons/api/mirage/config.js

```diff
@@ -368,36 +368,36 @@ export default function () {
         version: attrs.version,
       };
       // If adding host sets, push them into the array
-      if (method === 'add-host-sets') {
+      if (method === 'add-host-sources') {
         updatedAttrs.hostSetIds = target.hostSetIds;
```

Comment:
updatedAttrs.hostSetIds = target.hostSetIds --> updatedAttrs.hostSourcesIds = target.hostSourceIds

Reply:
Polymorphism here is difficult, so I'm deferring it.  That means that in the mocks, the relationship is target -> hostSet only for now.  In the future, this would become hostSource: Host || HostSet.